modular_skyrat/modules/hyposprays/code/hypospray_kits.dm

```diff
@@ -35,7 +33,7 @@
 	. = ..()
 	if(!length(case_designs))
 		populate_case_designs()
-	atom_storage.max_slots = 7
+	atom_storage.max_slots = 12
```

Comment:
You removed the pocket storage creep then almost doubled it's capacity?

Reply:
This is on Bubber. The storage was reduced to 7 on nova to compensate for the pocket thing.

Reply:
Why does it need 12 slots instead of the usual 7 every other box of anything has?

Reply:
Twelve slots is what we currently have. I didn't increase it, I reverted Nova's nerf.